sub/source/docs/introduction.rst

```diff
@@ -4,15 +4,36 @@
 Introduction to Sub
 ===================
 
-**UNDER CONSTRUCTION**
+Sub is an advanced open-source autopilot system for submersible ROVs (Remote Operated Vehicle) that supports multiple vehicle configurations. It offers a variety of operating modes from fully manual to fully autonomous.
 
+.. image:: ../images/sub-system.jpg
+    :target: ../_images/sub-system.jpg
 
 
-Vehicle Configuration Types
-===========================
-
+Vehicle Configuration Options
+=============================
 
+* Sub supports both brushed and brushless, uni-directional or bi-directional propulsion motors/thrusters
```

Comment:
I'm not quite sure we support uni-directional motors. I have never tried though...
Same for brushed, but I assume that is just a different MOT_PWM_TYPE?

Reply:
no, brushed vs brushless is just ESC type....both are PWM...could be DSHOT also, but I don't know of a brushed DSHOT ESC